packages/cashier/src/components/recent-transaction/recent-transaction.tsx

```diff
@@ -1,137 +0,0 @@
-import React from 'react';
-import { ButtonLink, Text } from '@deriv/components';
-import { useCryptoTransactions } from '@deriv/hooks';
-import { epochToMoment } from '@deriv/shared';
-import { observer, useStore } from '@deriv/stores';
-import { Localize } from '@deriv/translations';
-import classNames from 'classnames';
-import { getStatus } from '../../constants/transaction-status';
-import { useCashierStore } from '../../stores/useCashierStores';
-import './recent-transaction.scss';
-
-/** @deprecated Use `CryptoTransactionsSideNoteRecentTransaction` instead. */
-const RecentTransaction = observer(() => {
```

Comment:
The last use was replaced with `CryptoTransactionsSideNoteRecentTransaction` in this PR, Hence removing the `deprecated` component.